themes/login-forms/templates/AppHeader.ss

```diff
@@ -1,8 +1,8 @@
 <header class="app-brand">
     <a
         class="app-brand__link"
-        href="/"
-        title="Go back to homepage of <% if not $SiteConfig.Title %>$SiteConfig.Title<% else %>site<% end_if %>"
+        href="$BaseURL"
+        title="<%t SilverStripe\LoginForms.BackToHomePage "Go back to homepage" %>"
```

Comment:
I simplified the title here because I don't think we can have a conditional in a `<%t >` block.

If we really want to retrain this conditional wording, we'll need to have an `<% if > <% else %> <% end_if %>` block with two different translation strings.